packages/website/pages/pricing.js

```diff
@@ -151,8 +151,8 @@ const PricingTiers = () => (
               title="Expert"
               price="$10"
               isBestValue={true}
-              storageAllocation="60GiB storage"
-              storageOverageRate="+ $0.17/mo per additional GiB"
+              storageAllocation="120GiB storage"
+              storageOverageRate="+ $0.075/mo per additional GiB"
```

Comment:
According to [reqs](https://www.notion.so/pricing-page-change-tier-details-to-reflect-new-storage-prices-b3ad9e1d44784613ac6c9927cce36b5c) it seems like this should be $.08/mo ?